.prettierrc

```diff
@@ -0,0 +1,13 @@
+{
+  "prettier.singleQuote": true,
+  "prettier.arrowParens": "always",
+  "prettier.bracketSpacing": true,
+  "prettier.jsxSingleQuote": false,
+  "prettier.printWidth": 80,
+  "prettier.tabWidth": 2,
+  "prettier.useTabs": false,
+  "prettier.semi": true,
+  "prettier.endOfLine": "lf",
+  "prettier.singleAttributePerLine": false,
+  "prettier.bracketSameLine": false
+}
```

Comment:
According to prettier documentations, for file named as `.prettierrc` the content format should be `YAML` not `JSON`, or we can change file name to `.prettierrc.json` if want to be `JSON` format, and not includes prefix `prettier.` for each properties.

> prefix `prettier.` is an global prettier vscode extension configuration.

![image](https://user-images.githubusercontent.com/56349605/206654926-f338dfc4-37ea-4e39-aa67-d8b50afe46a8.png)